tests/tests/test_cluster_form.py

```diff
@@ -5,7 +5,7 @@
 from django.test import TestCase
 from tests.models import Band, BandMember, Album, Restaurant, Article, Author, Document, Gallery
 from modelcluster.forms import ClusterForm
-from django.forms import Textarea, CharField, ModelForm
+from django.forms import Textarea, CharField
```

Comment:
`ModelForm` was imported but not used -maybe you prefer another PR for that, let me know